core/build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
   alias(libs.plugins.kotlin.jvm)
 }
 
+repositories { maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots") } }
```

Comment:
Isn't this available here?
https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on

Reply:
This is for SNAPSHOT wallet SDK. I'll merge Wallet SDK changes and release new version before merging this PR (so will remove snapshot repo)